drivers/hmis/spec/requests/hmis/project_spec.rb

```diff
@@ -240,6 +231,52 @@
       expect(result.dig('data', 'projects', 'nodes').count).to eq(0)
     end
   end
+
+  describe 'canProjectAcceptReferral query' do
+    let(:can_accept_referral) do
+      <<~GRAPHQL
+        query GetCanProjectAcceptReferral($projectId: ID!, $referrerEnrollmentId: ID!) {
+          canProjectAcceptReferral(projectId:$projectId, referrerEnrollmentId:$referrerEnrollmentId)
+        }
+      GRAPHQL
+    end
+
+    let!(:p2) { create :hmis_hud_project, data_source: ds1, organization: o1, user: u1 }
+    let!(:referral_instance) { create :hmis_form_instance, role: :REFERRAL, entity: p2 }
+
+    let!(:c2) { create :hmis_hud_client, data_source: ds1 }
+
+    # p1 is the referring project
+    let!(:p1_ac) { create_access_control(hmis_user, p1) }
```

Comment:
I would say naming the vars "source_project"/"destination_project"  (or sending/receiving) is clearer than using the p1/p2 syntax. (I know I am guilty of this a lot😬 ).